2022-04-06 11:28:04,226 INFO    MainThread:29175 [wandb_setup.py:_flush():75] Loading settings from /home/ltorres/.config/wandb/settings
2022-04-06 11:28:04,226 INFO    MainThread:29175 [wandb_setup.py:_flush():75] Loading settings from /home/ltorres/leo/tesis/cloud-classification/code/wandb/settings
2022-04-06 11:28:04,226 INFO    MainThread:29175 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-04-06 11:28:04,226 INFO    MainThread:29175 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-04-06 11:28:04,226 INFO    MainThread:29175 [wandb_init.py:_log_setup():405] Logging user logs to /home/ltorres/leo/tesis/cloud-classification/code/wandb/run-20220406_112804-3dindk1y/logs/debug.log
2022-04-06 11:28:04,226 INFO    MainThread:29175 [wandb_init.py:_log_setup():406] Logging internal logs to /home/ltorres/leo/tesis/cloud-classification/code/wandb/run-20220406_112804-3dindk1y/logs/debug-internal.log
2022-04-06 11:28:04,226 INFO    MainThread:29175 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7ff1464a4580>
2022-04-06 11:28:04,227 INFO    MainThread:29175 [wandb_init.py:init():439] calling init triggers
2022-04-06 11:28:04,227 INFO    MainThread:29175 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0003, 'architecture': 'GATConvGNN', 'loss': 'LogeLoss', 'optim': 'SGD', 'dataset': 'GCD', 'builder': 'Multihead Attention', 'epochs': 15}
2022-04-06 11:28:04,227 INFO    MainThread:29175 [wandb_init.py:init():492] starting backend
2022-04-06 11:28:04,227 INFO    MainThread:29175 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-06 11:28:04,262 INFO    MainThread:29175 [backend.py:ensure_launched():219] starting backend process...
2022-04-06 11:28:04,291 INFO    MainThread:29175 [backend.py:ensure_launched():224] started backend process with pid: 29218
2022-04-06 11:28:04,293 INFO    MainThread:29175 [wandb_init.py:init():501] backend started and connected
2022-04-06 11:28:04,309 INFO    MainThread:29175 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-04-06 11:28:04,309 INFO    MainThread:29175 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-04-06 11:28:04,309 INFO    MainThread:29175 [wandb_init.py:init():565] updated telemetry
2022-04-06 11:28:04,394 INFO    MainThread:29175 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-04-06 11:28:05,174 INFO    MainThread:29175 [wandb_run.py:_on_init():1759] communicating current version
2022-04-06 11:28:05,259 INFO    MainThread:29175 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.12.12 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-06 11:28:05,259 INFO    MainThread:29175 [wandb_init.py:init():625] starting run threads in backend
2022-04-06 11:28:06,770 INFO    MainThread:29175 [wandb_run.py:_console_start():1733] atexit reg
2022-04-06 11:28:06,771 INFO    MainThread:29175 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-04-06 11:28:06,772 INFO    MainThread:29175 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-04-06 11:28:06,773 INFO    MainThread:29175 [wandb_run.py:_redirect():1667] Redirects installed.
2022-04-06 11:28:06,774 INFO    MainThread:29175 [wandb_init.py:init():664] run started, returning control to user process
2022-04-06 11:28:06,784 INFO    MainThread:29175 [wandb_init.py:_pause_backend():318] pausing backend
2022-04-06 11:28:06,784 INFO    MainThread:29175 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-04-06 11:28:06,819 INFO    MainThread:29175 [wandb_init.py:_resume_backend():327] resuming backend
2022-04-06 12:00:38,636 INFO    MainThread:29175 [wandb_init.py:_pause_backend():318] pausing backend
2022-04-06 12:00:38,637 INFO    MainThread:29175 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-04-06 12:00:38,856 INFO    MainThread:29175 [wandb_init.py:_resume_backend():327] resuming backend
2022-04-06 12:00:38,857 INFO    MainThread:29175 [wandb_run.py:finish():1539] finishing run leonardtd/cloud classification - adjacency/3dindk1y
2022-04-06 12:00:38,857 INFO    MainThread:29175 [jupyter.py:save_history():429] not saving jupyter history
2022-04-06 12:00:38,857 INFO    MainThread:29175 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-04-06 12:00:38,857 INFO    MainThread:29175 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-04-06 12:00:38,857 INFO    MainThread:29175 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-04-06 12:00:38,858 INFO    MainThread:29175 [wandb_run.py:_restore():1674] restore
2022-04-06 12:00:39,395 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2022-04-06 12:00:39,804 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2022-04-06 12:00:40,232 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 15840
}

2022-04-06 12:00:40,336 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 15840
}

2022-04-06 12:00:40,438 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 15840
}

2022-04-06 12:00:40,541 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 15840
}

2022-04-06 12:00:40,643 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 15840
}

2022-04-06 12:00:40,745 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 9995
  total_bytes: 15840
}

2022-04-06 12:00:40,850 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15171
  total_bytes: 15840
}

2022-04-06 12:00:40,954 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15171
  total_bytes: 15840
}

2022-04-06 12:00:41,056 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15840
  total_bytes: 15840
}

2022-04-06 12:00:41,160 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15840
  total_bytes: 15840
}

2022-04-06 12:00:41,263 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15840
  total_bytes: 15840
}

2022-04-06 12:00:41,366 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15840
  total_bytes: 15840
}

2022-04-06 12:00:41,468 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15840
  total_bytes: 15840
}

2022-04-06 12:00:41,571 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15840
  total_bytes: 15840
}

2022-04-06 12:00:42,497 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15840
  total_bytes: 15840
}

2022-04-06 12:00:43,132 INFO    MainThread:29175 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15840
  total_bytes: 15840
}
local_info {
}

2022-04-06 12:00:44,291 INFO    MainThread:29175 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-04-06 12:00:44,292 INFO    MainThread:29175 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-04-06 12:00:44,298 INFO    MainThread:29175 [wandb_run.py:_footer_sync_info():2822] logging synced files
